pages/osx/xctool.md

```diff
@@ -0,0 +1,15 @@
+# XCTool
+
+> Tool for build Xcode projects
+
+- Build a project that is part of a workspace
+
+`xctool -workspace {{YourWorkspace.xcworkspace}} -scheme {{YourScheme}} build`
+
+- Build a single project without any workspace
+
+`xctool.sh -project {{YourProject.xcodeproj}} -scheme {{YourScheme}} build`
```

Comment:
This seems like it is a simpler example than the previous one. If that's the case, I'd suggest swapping the order, so that people can read this going from simple commands to (slightly) more complex ones.